flocker/node/agents/functional/test_ebs.py

```diff
@@ -66,6 +69,61 @@ class EBSBlockDeviceAPIInterfaceTests(
     """
     Interface adherence Tests for ``EBSBlockDeviceAPI``.
     """
+    @capture_logging(None)
+    def test_volume_busy_error_on_busy_state(self, logger):
+        """
+        A ``VolumeBusy`` is raised if a volume's attachment state is "busy"
+        when attempting to detach the volume. This can occur if an attempt
+        is made to detach a volume that has not been unmounted.
+        """
+        try:
+            config = get_blockdevice_config(ProviderType.aws)
+        except InvalidConfig as e:
+            self.skipTest(str(e))
```

Comment:
:question: This snippet is repeated quite a lot. Consider extracting a helper function that does this check, e.g. `require_aws_blockdevice_config`.